topics/contributing/tutorials/github-contribution/tutorial.md

```diff
@@ -152,6 +227,31 @@ You have created your first branch! Now you want to make the change in the `CONT
 
 ![Divergence of the branch compared to main](../../images/PR_branches_02.png "The changes on your branch will not be on the main branch")
 
+<div class="Desktop" markdown="1">
```

Comment:
As before, this section does not show correctly when I preview the page in GitHub. I assume it worked fine in your dev environment. 
<img width="1278" alt="image" src="https://github.com/user-attachments/assets/93f8ccf0-8b6f-4bf2-9abf-8e028588a712" />